hw/top_earlgrey/cdc/cdc_waivers.alert_handler.tcl

```diff
@@ -10,3 +10,4 @@
 
 set_rule_status -rule {W_CNTL} -status {Waived} -expression {(Signal=~"*u_alert_handler.gen_alerts*u_secure_anchor_flop*") && (ReceivingFlop=~"*u_*alert_sender*.u_decode_ack*")} -comment {Alert ACK remains high until alert sender acked}
 set_rule_status -rule {W_CNTL} -status {Waived} -expression {(Signal=~"*u_alert_handler.gen_alerts*u_secure_anchor_flop*") && (ReceivingFlop=~"*u_*alert_sender*.u_decode_ping*")} -comment {Alert PING remains high until alert sender acked}
+set_rule_status -rule {W_RECON_GROUPS} -status {Waived} -expression {(ControlSignal=~"*u_alert_handler.u_alert_handler_lpg_ctrl.gen_lpgs**") && (ReconSignal=~"*u_alert_handler.u_alert_handler_lpg_ctrl.gen_alert_map*unused_logic*")} -comment {Intended logic to avoid lint error}
```

Comment:
does this mean this isn't actually real logic, but just unused signals we added to avoid the ascentlint `NOT READ` error?

Reply:
Yes, it's found in https://github.com/lowRISC/opentitan/blob/master/hw/ip_templates/alert_handler/rtl/alert_handler_lpg_ctrl.sv#L74-L88

Reply:
sounds good, thank you